spk/rutorrent/Makefile

```diff
@@ -14,7 +14,14 @@ ADMIN_URL = /rutorrent/
 RELOAD_UI = yes
 STARTABLE = yes
 DISPLAY_NAME = ruTorrent
-CHANGELOG ="1. Update rutorrent to 3.10<br/>2. Update to rtorrent 0.9.8<br/>3. Update to libtorrent 0.13.8<br/>4. Update to xmlrpc-c 1.51.06<br/>5. Update openssl to 1.1"
+CHANGELOG = "1. Update rutorrent to 3.10<br/>
+CHANGELOG += 2. Update to rtorrent 0.9.8<br/>
+CHANGELOG += 3. Update to libtorrent 0.13.8<br/>
+CHANGELOG += 4. Update to xmlrpc-c 1.51.06<br/>
+CHANGELOG += 5. Update openssl to 1.1<br/>
+CHANGELOG += 6. Embed cloudflare requirements (requires python 3)<br/>
+CHANGELOG += 7. Embed pgrep (required by tasks)<br/>
+CHANGELOG += 8. Fixed stat<br />"
```

Comment:
Sorry but as this changelog targets end-user audience, I would go into so much details - whereas I am pleased to find in it commit message.

Reply:
I changed the changelog but I must confess I didn't find anything really satisfactory to put in there (fixing ? :/ )